sycl/include/CL/sycl/detail/backend_traits_opencl.hpp

```diff
@@ -188,6 +189,28 @@ template <> struct InteropFeatureSupportMap<backend::opencl> {
   static constexpr bool MakeKernel = true;
   static constexpr bool MakeKernelBundle = true;
 };
+
+namespace pi {
+// Cast for std::vector<cl_event>, according to the spec, make_event
+// should create one(?) event from a vector of cl_event
+template <class To> inline To cast(std::vector<cl_event> value) {
+  RT::assertion(value.size() == 1,
+                "Temporary workaround requires that the "
+                "size of the input vector for make_event be equal to one.");
+  return (To)(value[0]);
+}
+
+// These conversions should use PI interop API.
+template <> inline PiProgram cast(cl_program) {
+  RT::assertion(false, "pi::cast -> use piextCreateProgramWithNativeHandle");
```

Comment:
`static_assert` here as well?

Reply:
As per offline discussion `delete` seems to be a good solution to this.